golang/go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/libp2p/go-libp2p v0.14.4
 	github.com/libp2p/go-libp2p-core v0.8.6
 	github.com/libp2p/go-libp2p-noise v0.2.0
+	github.com/libp2p/go-libp2p-quic-transport v0.10.0 // indirect
```

Comment:
Unfortunately I could not upgrade it to the newest version (0.15.0). Would you do that on your own?